ui/src/components/HomePage/PopularGridSection.jsx

```
@@ -8,34 +8,38 @@ import { Card, CardContent, CardHeader, CardDescription, CardTitle, CardFooter }
-            <Button>Explore More</Button>
-            <Button variant={'outline'}>Download Now</Button>
-          </CardFooter>
-        </Card>
-        ))
-      }
-    </div> );
+    <section className="grid grid-cols max-w-7xl mx-auto w-full gap-6">
+      <h2 className="text-5xl font-bold px-4">Popular Products</h2>
+      <div className='grid gap-4 grid-cols-[repeat(auto-fit,minmax(305px,1fr))]'>
+        {products.map((prod) => (
+          <Card className='grid grid-cols max-w-sm grid-cols-subgrid overflow-hidden'>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)